.buildconfig.yml

```diff
@@ -328,4 +328,3 @@ projects:
     path: samples/dataprotect
     description: 'An app demoing how to use the Dataprotect component to load and store encrypted data in SharedPreferences.'
     publish: false
-
```

Comment:
Heisen-line shows up locally but disappears here 😠

Reply:
Nevermind, there are just two extra lines at the end of buildconfig right now on master.